src/reusecore/Search/searchbox.style.js

```diff
@@ -15,6 +15,10 @@ export const SearchWrapper = styled.div`
       border-radius: 5px;
       border: 1px solid ${(props) => props.theme.headingColor};
       background-color: ${(props) => props.theme.shadowLightColor};
+      
+      /*Changes color for the text*/
+      color: ${(props) => props.theme.textColor};
```

Comment:
Will this affect the search field on /blog, too?

Hopefully, it does as that field is in need of similar dark mode treatment.

Reply:
I see now that it does.